libs/RobotKit/include/StateMachine.h

```diff
@@ -43,6 +46,20 @@ namespace sm::guard {
 
 }	// namespace sm::guard
 
+namespace sm::action {
+
+	using irc = interface::RobotController;
+
+	struct start_goToSleep_timeout {
+		auto operator()(irc &rc) const { rc.startGoToSleepTimeout(); }
+	};
+
+	struct stop_goToSleep_timeout {
+		auto operator()(irc &rc) const { rc.stopGoToSleepTimeout(); }
+	};
```

Comment:
après réflexion, je reviens un peu sur ce que j'avais dit, cf mon commentaire de review.

```suggestion
	struct start_sleep_timeout {
		auto operator()(irc &rc) const { rc.startGoToSleepTimeout(); }
	};

	struct stop_sleep_timeout {
		auto operator()(irc &rc) const { rc.stopGoToSleepTimeout(); }
	};
```